testlib/infra.py

```diff
@@ -43,6 +43,11 @@ def clean_up():
 
     error_strings = []
 
+    terminate_traces(MONITOR_DBUS_SIGNALS)
+
+    if process_exists("stratisd") is None:
+        raise RuntimeError("stratisd process is not running")
+
```

Comment:
Plz move ```error_strings = []``` to right before the definition of ```check_result()```. After that, we can consider this complete.

Reply:
Resolved.